react-hs-examples/redux/stackjs.yaml

```diff
@@ -0,0 +1,17 @@
+resolver: lts-7.19
+compiler: ghcjs-0.2.1.9007019_ghc-8.0.1
```

Comment:
could you change the file names like this?:

```
stack.yaml => stack-no-js.yaml
stackjs.yaml => stack.yaml
```

(I don't care so much about the specific names, I'd just like to keep ghcjs the default.)